src/components/SelectButton.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { RaisedButton } from 'material-ui';
+import SvgIcon from 'material-ui/SvgIcon';
+
+import * as colors from '../colors';
+
+export const UNSELECTED = 'ADD TO MY COLLECTION';
```

Comment:
Actually, I see why you are putting them as variables, but they definitely shouldn't be exported. Let's figure out how to restructure this code and where component boundaries should be.